packages/framework/src/utils/options.utils.ts

```diff
@@ -5,3 +5,15 @@ export function resolveApiUrl(providedApiUrl?: string): string {
 export function resolveSecretKey(providedSecretKey?: string): string {
   return providedSecretKey || process.env.NOVU_SECRET_KEY || process.env.NOVU_API_KEY || '';
 }
+
+export function resolveLogging(providedLogging?: boolean): boolean {
+  if (providedLogging !== undefined) {
+    return providedLogging;
+  }
+  if (process.env.NOVU_LOGGING !== undefined) {
+    return process.env.NOVU_LOGGING === 'true';
+  }
+
+  // Disable verbose logging in test and production environments
+  return !['test', 'production'].includes(process.env.NODE_ENV);
```

Comment:
Love it! 
But can't we reuse NOVU_LOGGING to toggle this? so for test and prod it will be false, but in case we want to debug some edge cases, we could turn it on in prod. 
in addition, we could use more specific env variables so they won't collide in the future in any way FRAMEWORK_LOGGING, although it should not be an issue.